source/appModules/soffice.py

```diff
@@ -449,13 +457,36 @@ class SymphonyDocument(CompoundDocument):
 	# variables used for handling announcements resulting from gestures
 	GESTURE_ANNOUNCEMENT_TIMEOUT = 2.0
 	announceFormattingGestureChange = False
+	formattingGestureObjectIds = []
 	lastFormattingGestureEventTime = 0
 
 	@staticmethod
-	def isFormattingChangeAnnouncementEnabled() -> bool:
-		return SymphonyDocument.announceFormattingGestureChange and time.time() < (
+	def isFormattingChangeAnnouncementEnabled(obj: NVDAObject) -> bool:
```

Comment:
this is an API breaking change, can it please be documented?

Reply:
This method was only introduced recently in 20bdb39e6f83ef87726817cb1bad31fa7b0f73cb, i.e. if I understand correctly, it's only an API change within the current development version, not as compared to any released NVDA version. Should it still be documented?

Reply:
Right, never mind